syntax = "proto3";

package jacobs_proto;

service jacobs_service {

 rpc test_message(message_request) returns (message_response) {}

}

message message_request {  // you can add comments like this
 string message = 1;
 string requester = 2;
}

/* or add comments like this */

message message_response {
 string message = 1;
 string sender = 2;
}